modules/apps/frontend-js/frontend-js-tooltip-support-web/src/main/resources/META-INF/resources/index.tsx

```diff
@@ -54,7 +54,12 @@ const getDefaultTooltipContainer = () => {
 export default () => {
 	render(
 		ClayTooltipProvider,
-		{scope: SELECTOR_TRIGGER},
+		{
+			containerProps: {
+				className: 'cadmin',
```

Comment:
<strike>When using this way, We're just appending `cadmin` class in the current ClayTooltip instance?</strike>

[Nope, It will just add `cadmin` to ClayPortal component](https://github.com/liferay/clay/blob/master/packages/clay-tooltip/src/TooltipProvider.tsx#L143)

Reply:
@matuzalemsteles @julien, when taking a look on [ClayTooltipProvider API doc](https://clayui.com/docs/components/tooltip/api.html)s the description is not being shown with the current JSDoc. Is this expected?

Reply:
It probably wouldn't be expected 🙂, but this may be related to the react-docgen bug.